css/_uswds-theme-custom-styles.scss

```diff
@@ -3926,3 +3926,155 @@ $do-dont-top-bar-width: 1;
     background-color: color("red-50");
   }
 }
+
+$proposal-phase-color: color("yellow-20v");
+$accepted-phase-color: color("yellow-20v");
+$rejected-phase-color: color("gray-cool-60");
+$development-phase-color: color("blue-20v");
+$experimental-phase-color: color("mint-20");
+$released-phase-color: color("mint-50");
+$caution-phase-color: color("orange-30v");
+$deprecated-phase-color: color("red-warm-60v");
+
+.lifecycle {
+  &-border-bottom {
+    border-bottom: units(1) solid;
+  }
+
+  &-border--proposal {
+    border-color: $proposal-phase-color !important;
+  }
+
+  &-border--development {
+    border-color: $development-phase-color !important;
+  }
+
+  &-border--released {
+    border-color: $released-phase-color !important;
+  }
+
+  &-border--deprecated {
+    border-color: $deprecated-phase-color !important;
+  }
+
+  &-bg--proposal {
+    background-color: $proposal-phase-color !important;
+  }
+
+  &-bg--development {
+    background-color: $development-phase-color !important;
+  }
+
+  &-bg--released {
+    background-color: $released-phase-color !important;
+    color: white !important;
+  }
+
+  &-bg--experimental {
+    background-color: $experimental-phase-color !important;
+  }
+
+  &-bg--caution {
+    background-color: $caution-phase-color !important;
+  }
+
+  &-bg--deprecated {
+    background-color: $deprecated-phase-color !important;
+    color: white !important;
+  }
+
+  &-bg--accepted {
+    background-color: $accepted-phase-color !important;
+  }
+
+  &-bg--rejected {
+    background-color: $rejected-phase-color !important;
+    color: white !important;
+  }
+}
+
+.lifecycle-process {
+  .usa-process-list__item:last-child {
+    padding-bottom: units(1);
+  }
+}
+
+.lifecycle-phase {
+  @include u-measure(6);
+
+  &__intro {
+    @include at-media("desktop-lg") {
+      @include grid-col(3);
+      @include u-margin-right(4);
+    }
+  }
+
+  &__heading {
+    @include u-line-height("sans", 1);
+    @include u-margin-top(0);
+    @include u-margin-bottom(1);
+  }
+
+  &__description {
+    @include u-margin-top(0);
+  }
+
+  &__subphase {
+    @include u-margin-bottom(4);
+    @include u-measure(5);
+
+    p {
+      @include u-margin-y("105");
+    }
+
+    ul,
+    li {
+      @include u-line-height("sans", 4);
+    }
+
+    &:last-of-type {
+      margin-bottom: 0;
+    }
+  }
+
+  &__subphase-header {
+    @include u-display("inline-block");
+    @include u-margin-bottom(0);
+    @include u-padding-y("05");
+    border-bottom: 4px solid color("gray-30");
+
+    &:first-of-type {
+      margin-top: 0;
+    }
+  }
+}
+
+.usa-prose > .lifecycle-table {
+  @include u-font("lang", "3xs");
```

Comment:
This is pretty small, but it looks like other tables share small font size too.

Ex: `/utilities/`.